commands/active_test.go

```diff
@@ -0,0 +1,51 @@
+package commands
+
+import (
+	"os"
+	"testing"
+
+	"github.com/docker/machine/drivers/fakedriver"
+	"github.com/docker/machine/libmachine/host"
+	"github.com/docker/machine/libmachine/state"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	h = host.Host{
+		Name:       "foo",
+		DriverName: "fakedriver",
+		Driver: &fakedriver.Driver{
+			MockState: state.Running,
+		},
+	}
+)
+
+func TestIsActiveReturnsFalseGivenNoActiveHost(t *testing.T) {
```

Comment:
Why not make this into just one test with a test table?  You may have seen me ~~pestering~~ commenting about it on some of the other issues. e.g.:

```go
	cases := []struct {
		dockerHost     string
		currentState   state.State
		url            string
		expectedResult bool
		expectedErr    error
	}{
		{"", state.Running, "tcp://1.2.3.4:2376", false, nil},
		{"tcp://1.2.3.4:2376", state.Stopped, "tcp://1.2.3.4:2376", false, nil},
	}

	for _, c := range cases {
		result, err := isActive(&h, c.currentState, c.url)
		assert.Equal(t, expectedResult.result)
		assert.Equal(t, expectedErr, err)
	}
```

Please make sure to check the error too.